package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@neuralegion/nexploit-cli",
-  "version": "4.0.0",
+  "version": "0.0.0",
```

Comment:
@ArtemDerevnjuk is that intentional?

Reply:
Yeah, semantic-release sets it during a build.